types/aggregator.go

```diff
@@ -27,6 +28,7 @@ type Aggregators struct {
 // CreateAggregator request payload for creating a new aggregator.
 type CreateAggregator struct {
 	Name                    string   `json:"name"`
+	EnvironmentID           string   `json:"environment_id"`
```

Comment:
I propose passing a Environment string, validate is unique and save the id in the database.